integration-tests/xml/src/main/java/org/apache/camel/quarkus/component/xml/it/XmlRouteBuilder.java

```diff
@@ -18,9 +18,17 @@
 
 import org.w3c.dom.Document;
 
+import io.quarkus.runtime.annotations.RegisterForReflection;
 import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.xslt.saxon.XsltSaxonAggregationStrategy;
 import org.apache.camel.support.builder.Namespaces;
 
+@RegisterForReflection(classNames = {
+        "net.sf.saxon.Configuration",
+        "net.sf.saxon.functions.String_1",
+        "net.sf.saxon.functions.Tokenize_1",
+        "net.sf.saxon.functions.StringJoin",
+        "org.apache.camel.component.xslt.saxon.XsltSaxonBuilder" })
```

Comment:
I see that camel-quarkus-xslt-saxon is JVM only. 

If porting camel-quarkus-xslt-saxon to native would be too much work, then it would be worthwhile to put a comment here saying that `@RegisterForReflection` should be removed with fixing https://github.com/apache/camel-quarkus/issues/1615

In https://github.com/apache/camel-quarkus/issues/1615, we should note that XmlRouteBuilder.java needs some care

Reply:
yeah, I will leave a comment here.